backend/inspirehep/pidstore/minters/cnum.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+from inspirehep.pidstore.minters.base import Minter
+from inspirehep.pidstore.providers.cnum import InspireCNUMProvider
+
+
+class CNUMMinter(Minter):
+
+    pid_value_path = "cnum"
+    pid_type = "cnum"
+    provider = InspireCNUMProvider
+
+    @classmethod
+    def mint(cls, object_uuid, data):
+        minter = cls(object_uuid, data)
+        minter.validate()
+        if "cnum" not in data:
```

Comment:
what about existing CNUMs coming from legacy in the migration? are you registering those?

Reply:
Nope, this is only for minting new conference records. The migration is not part of this task

Reply:
is it part of any task? if not I believe it should be part of this one, otherwise you'll register existing CNUMs for conferences.

Reply:
Agree on migrating conferences to avoid overwriting CNUMs, not sure is part of this though, since it won't run unless we create conferences record through submissions. I'll create another subtask for this User Story, WDYT?

Reply:
After discussing IRL, we decided to have some checks here. Some edge cases won't be handled, as describer in the `CNUMMinter` documentation.